tools/python/util/onnx_model_utils.py

```diff
@@ -103,12 +104,25 @@ def optimize_model(
     :param level: onnxruntime.GraphOptimizationLevel to use. Default is ORT_ENABLE_BASIC.
     :param log_level: Log level. Defaults to Error (3) so we don't get output about unused initializers being removed.
                       Warning (2) or Info (1) may be desirable in some scenarios.
+    :param use_external_initializers: Set flag to write initializers to an external file. Required if model > 2GB.
+                                      Requires onnxruntime 1.17+
     """
     so = ort.SessionOptions()
     so.optimized_model_filepath = str(output_path.resolve())
     so.graph_optimization_level = level
     so.log_severity_level = log_level
 
+    # save using external initializers so models > 2 GB are handled
+    if use_external_initializers:
+        major, minor, rest = ort.__version__.split(".", 3)
+        if int(major) > 1 or (int(major) == 1 and int(minor) >= 17):
```

Comment:
```suggestion
        if (int(major), int(minor)) >= (1, 17):
```
side question: when is ORT 2.0 coming out? :)

Reply:
Dunno - but I'm ready! :P

Did not know you could do tuple comparison like this!